src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
Don't use static methods in your solution

Reply:
I will change it but I thought its better to use static method since we do not need to create a new specific object of ColorSupplier as we this method will be same for any of those so we can just call it from our class. please explain me why I am wrong

Reply:
done